src/Raven.Server/Utils/MicrosoftLogging/WebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+using Raven.Server.Config.Categories;
+using Sparrow;
+using Sparrow.Logging;
+
+namespace Raven.Server.Utils.MicrosoftLogging;
+
+public static class WebHostBuilderExtensions
+{
+    public static IWebHostBuilder ConfigureMicrosoftLogging(this IWebHostBuilder hostBuilder, ref LoggingSource loggingSource, LogsConfiguration configurationLogs)
+    {
+        if (configurationLogs.DisableMicrosoftLogs)
+            return hostBuilder;
+        
+        string logPath = configurationLogs.MicrosoftLogsPath.FullPath;
+        var internalLoggingSource = new LoggingSource(LogMode.None, logPath, "Microsoft Log", TimeSpan.MaxValue, long.MaxValue);
+        internalLoggingSource.MaxFileSizeInBytes = configurationLogs.MicrosoftLogsMaxFileSize.GetValue(SizeUnit.Bytes);
```

Comment:
I would prefer to have:
`(configurationLogs.MicrosoftLogsMaxFileSize ?? configurationLogs.MaxFileSize)`
same for other options

We want to have same conf as for regular logs, but with the ability to override if neeed

Reply:
Done